util/cron/nightly

```diff
@@ -248,7 +248,7 @@ $user = `whoami`;
 chomp($user);
 $debugmail = $ENV{'CHPL_NIGHTLY_DEBUG_EMAIL'};
 if ($debugmail eq "") {
-    $debugmail = "$user\@cray.com";
+    $debugmail = "$user\@hpe.com";
```

Comment:
Yeah, I realized this style of change was unlikely to "just work" but also seemed unlikely to be used much.  Do you have other thoughts about how to address this short of rewriting the script to accept a new environment variable or argument or something like that?  I suppose it might be best to just put an error message into this conditional?

Reply:
Yeah, I might just make it an error if `$debug == 1` and `CHPL_NIGHTLY_DEBUG_EMAIL` isn't set

Reply:
I did this for this and tokctnightly.